### DISCUSSION TITLE
Not updating

### DISCUSSION THREAD

**Discussion by Kmfernan5** [flowershow/flowershow#546] [General and Help]
see screenshots below. Thanks!

https://share.getcloudapp.com/E0uLB7g7
https://share.getcloudapp.com/NQupmzNK

**Comment by tommdq** [comment]
Same problem here!

**Comment by Giampy72** [comment]
Same problem .  Did you fix ?
If I check into content folder I see my files but nothing happen into web site. It always shows the welcome page. 
there is a specific file to upload such as home or index to make the site works and shows all my files ?

**Comment by rufuspollock** [comment]
@Giampy72 sorry we missed this when you first reported. Can you give a bit more detail - i took a quick look at the images but wasn't sure.

**Comment by everythingness** [comment]
same problem here! 
followed all the instructions and using vercel.
the github repository shows that the test page from obsidian has been pushed to github.
vercel shows the test page in the assets for the site.
but i still see the flowershow welcome page.
any further information i can give you?

**Reply by everythingness** [reply]
**ah never mind. i should have made an index file to replace the default one :)**
